src/content/effective-dart/usage.md

```diff
@@ -350,19 +374,27 @@ class UploadException {
 
   @override
   String toString() {
+    // ···
```

Comment:
@parlough I know I've asked about this before, so sorry for forgetting! But is there a way around these `// ...` that happen when you end and restart docregions? Also on line 358

Reply:
You should be able to add `plaster="none"` to the `code-excerpt` instruction.